oval-schemas/independent-definitions-schema.xsd

```diff
@@ -2491,6 +2656,94 @@ SERVERPROPERTY('IsClustered') AS [is_clustered]
                   </xsd:simpleType>
             </xsd:attribute>
       </xsd:complexType>
+      <xsd:complexType name="EntityObjectShellType">
+            <xsd:annotation>
+                  <xsd:documentation>The EntityObjectShellType complex type defines a string entity value that is restricted to a set of command shells. The empty string is also allowed to support empty elements associated with variable references.</xsd:documentation>
+            </xsd:annotation>
+            <xsd:simpleContent>
+                  <xsd:restriction base="oval-def:EntityObjectStringType">
+                        <xsd:enumeration value="sh">
+                              <xsd:annotation>
+                                    <xsd:documentation>The borne shell (sh)</xsd:documentation>
+                              </xsd:annotation>
+                        </xsd:enumeration>
+                        <xsd:enumeration value="bash">
+                              <xsd:annotation>
+                                    <xsd:documentation>The gnu borne again shell (bash).</xsd:documentation>
+                              </xsd:annotation>
+                        </xsd:enumeration>
+                        <xsd:enumeration value="csh">
+                              <xsd:annotation>
+                                    <xsd:documentation>The C shell (csh).</xsd:documentation>
+                              </xsd:annotation>
+                        </xsd:enumeration>
+                        <xsd:enumeration value="ksh">
+                              <xsd:annotation>
+                                    <xsd:documentation>The korn shell (ksh).</xsd:documentation>
+                              </xsd:annotation>
+                        </xsd:enumeration>
```

Comment:
Add zsh for sure, since that's the standard shell on macOS now.

Reply:
Good catch.  Will update